lib/src/extensions/matches_request.dart

```diff
@@ -50,6 +51,10 @@ extension MatchesRequest on RequestOptions {
       return true;
     }
 
+    /// if data is MockDataCallback do not need to match;
+    if (expected is MockDataCallback) {
+      return true;
+    }
```

Comment:
Sorry, I forgot to mention some little changes to make code better look.

Please change to:
```dart
//  If data is MockDataCallback do not need to match.
if (expected is MockDataCallback) return true
```